### PR TITLE
Add nginx ingress

### PR DESCRIPTION
**Pull request by ommkar-xl**

### CODE REVIEW

**Review by chandras-xl — APPROVED** [review]
#29 LGTM

File: values-nginx.yaml [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+ImagePullPolicy: "Always"
+
+## Secrets must be manually created in the namespace.
+# ImagePullSecret: xlRelease
+
+## Install nginx subchart. If you have nginx already installed, set 'install' to 'false'.
+## If you have any other ingress controller installed, you can set the 'install' to 'false'.
+## Ref: https://github.com/bitnami/charts/blob/master/bitnami/nginx-ingress-controller/README.md
+nginx-ingress-controller:
+  install: true
```
Comment (spaliwal-xl) [inline values-nginx.yaml:28]: Add image pull secret option for Nginx-ingress controller

Comment (ommkar-xl) [reply values-nginx.yaml:28]: changed were added

**Review by spaliwal-xl — APPROVED** [review]
#29 LGTM

**Pull request merged**